arduino/smartcar/smartcar.ino

```diff
@@ -102,19 +104,29 @@ void loop()
  * Used when connected to MQTT server.
  */
 void MQTTMessageInput(){
-    if (mqtt.connect("arduino", "public", "public")) {
+
+    if (mqtt.connect("arduino", "public", "public") && mqtt.connected()) {
```

Comment:
mqtt.connect and mqtt.connected, is the difference between the two that the first is only checking for the specific connection that's in the parameters?

Reply:
This change was redundant, the first is connecting and the second is checking if connected, if one is true, they both are. I have change that and will be included in the latest push.